app/helpers/dashboard_helper.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module DashboardHelper
+  def dashboard_cards_by_role
+    {
+      challenge_manager: [
+        { image_path: 'folder_open', href: 'manage_submissions', alt: 'submissions', title: 'Submissions',
+          subtitle: 'Manage submissions, evaluations, and evaluators.' },
```

Comment:
while you were out we decided to add the Challenges menu item linking to the Phoenix Portal app. can you add that option?
![Screenshot 2024-09-06 at 5 07 04 PM](https://github.com/user-attachments/assets/edd20ac3-3b0b-4050-a4ea-183f789e74c0)